.codegen/service.go.tmpl

```diff
@@ -19,20 +19,42 @@ import (
 {{end}}
 
 {{define "service"}}
-var Cmd = &cobra.Command{
-	Use:   "{{(.TrimPrefix "account").KebabName}}",
-	{{- if .Description }}
-	Short: `{{.Summary | without "`"}}`,
-	Long: `{{.Comment  "  " 80 | without "`"}}`,
-	{{- end }}
-	Annotations: map[string]string{
-		"package": "{{ .Package.Name }}",
-	},
-	{{- if .IsPrivatePreview }}
+// Slice with functions to override default command behavior.
+// Functions can be added from the `init()` function in manually curated files in this directory.
+var cmdOverrides []func(*cobra.Command)
```

Comment:
I assume we're generally comfortable with using `init()` from overrides as they (typically) don't have global state? What if an `init()` function does at some point depend on global state?

Reply:
Right, but the closures added to that list might capture global state as well. I didn't see any that had that issue, but I guess it couldn't be prevented?

Reply:
Indeed -- we need to do code review to prevent this. I don't think there is a way to programmatically prevent this.